### PR TITLE
Add Openable run action

### PR DESCRIPTION
**Pull request by rawagner**
Signed-off-by: Rastislav Wagner <rawagner@redhat.com>

### CODE REVIEW

**Comment by rawagner** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
@rawagner I tried to rerun the pr-check job and will whether test will fail again. In that case I would ask you to check the result and repair failing test, seems that they are connected with changes you made. 
Another thing is whether there should be at least some unit tests written, please add them or create another issue for that. Thanks!

**Comment by rawagner** [comment]
Added a few tests. All failing tests are failing on oxygen and some of them will be fixed by https://github.com/eclipse/reddeer/pull/1753

File: tests/org.eclipse.reddeer.jface.test/src/org/eclipse/reddeer/jface/test/window/EclipseWindow.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package org.eclipse.reddeer.jface.test.window;
```
Comment (odockal) [inline tests/org.eclipse.reddeer.jface.test/src/org/eclipse/reddeer/jface/test/window/EclipseWindow.java]: Add copyright, please.

Comment (odockal) [inline tests/org.eclipse.reddeer.jface.test/src/org/eclipse/reddeer/jface/test/window/EclipseWindow.java]: Counts also for other new classes.

Comment (rawagner) [inline tests/org.eclipse.reddeer.jface.test/src/org/eclipse/reddeer/jface/test/window/EclipseWindow.java]: added all the missing copyrights

**Pull request merged**